### DISCUSSION TITLE
OpenAPI block rendering is incomplete (especially examples)

### DISCUSSION THREAD

**Discussion by rocchisanijl** [GitbookIO/community#550] [4. Open API]
### Ask a question, start a conversation, or make an annoucement

the rendering of openapi is incomplete in particular for example put in the openapi file (see the screen shots comparing rendering gitbook versus swaggerhub)
![Capture d’écran du 2024-03-27 17-52-06](https://github.com/GitbookIO/community/assets/27018079/18bb4617-5add-4689-ab7b-7e402979e59b)
![Capture d’écran du 2024-03-27 17-52-29](https://github.com/GitbookIO/community/assets/27018079/7634cf40-a9c4-4fcf-8c74-5a79f8140c9d)
[swagger_e-financing_1.3.9.zip](https://github.com/GitbookIO/community/files/14776718/swagger_e-financing_1.3.9.zip)
https://docs.scalexpert.societegenerale.com/apidocs/3mLlrPx3sPtekcQvEEUg/api-reference/e-financing-api/uat-api-for-partners/v-1.3.9-e-financing-api-uat-for-partners-only

**Comment by addisonschultz** [comment]
Hey @rocchisanijl! Can you let us know more specifically what parts of the block aren't rendering as you'd expect?

**Reply by rocchisanijl** [reply]
the response of the endpoint /_simulate-solutions is not complete. As you can see in the screenshot from swagger hub.

**Reply by addisonschultz** [reply]
Thanks for pointing this out - I've logged it for the team to dive into!

**Comment by Vic-user** [comment]
Hello, @addisonschultz is there any progress on this topic? I'm encountering the same problem, where the example from my openapi.json file is not displayed in the response block of the layout.

**Comment by rocchisanijl** [comment]
Hi @Vic-user 
it seems it has been improved but not yet as expected. In the meantime I had manually some code block for examples.
![image](https://github.com/user-attachments/assets/98fd3c92-bc54-4165-826f-51e0d6fba92e)

**Reply by Vic-user** [reply]
Hi @rocchisanijl , how did you manage to insert the response example in the layout? Or did you maybe do some changes to the source file? When I import an OpenAPI file, it will automatically generate the layout, but it seems to omit the example section from my json file.
![image](https://github.com/user-attachments/assets/91ca5599-657b-4dfd-a430-63d26b7330d7)

**Reply by rocchisanijl** [reply]
Hi I just add some "code block" in the page below the open API block

![image](https://github.com/user-attachments/assets/a3839b3b-eb79-4cd3-af09-7541f4f579d3)

**Reply by Vic-user** [reply]
@rocchisanijl Thanks for your input, I've followed your suggestion as a temporary solution. Hopefully we get something better down the road for such scenarios.

**Comment by Vic-user** [comment]
Hi team, do we have any progress on that? It will be really useful to have the actual examples come out-of-the-box when an OAS is imported.
@addisonschultz can you share a link to the opened issue for visibility?
Regards,
Victor

**Reply by addisonschultz** [reply]
Hi @Vic-user - unfortunately not a lot of movement on this currently. We do have it logged, and I see some chats that we’ll be investigating this with some other improvements to the overall OpenAPI block in a coming sprint - but I don’t have any timelines right now!

**Comment by MantisClone** [comment]
Hello, any progress on this?
